tests/test_json_output.py

```diff
@@ -0,0 +1,51 @@
+from modules.Connection import Connection
+from BaseCollector import BaseCollector
+import pytest
+import json
+import yaml
+
+
+@pytest.fixture(scope='class')
+def bc_yaml():
```

Comment:
```suggestion
def rest_yaml():
```
would go as close as possible from the naming

Reply:
good Idea.